linebot/client_test.go

```diff
@@ -25,28 +25,50 @@ import (
 )
 
 func mockClient(server *httptest.Server) (*Client, error) {
-	client, err := New(
-		"testsecret",
-		"testtoken",
-		WithHTTPClient(&http.Client{
+	u, err := url.ParseRequestURI(server.URL)
```

Comment:
I think it is enough to use same client having both of endpointBase and endpointBaseData, checking which endpoint is used inside mock server's handler

Reply:
Changes are applied in a4d28fb and 19b9b7c. Later one is to make those changes introduced in #172 follow the latest method signature of `mockClient`.